api/CanvasCaptureMediaStream.json

```diff
@@ -0,0 +1,162 @@
+          },
+          "opera_android": {
+            "version_added": false
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "canvas": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/CanvasCaptureMediaStream/canvas",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "41",
+              "notes": "Disabled by default; set the preference <code>canvas.capturestream.enabled</code> to <code>true</code> to activate."
+            },
+            "firefox_android": {
+              "version_added": "41",
+              "notes": "Disabled by default; set the preference <code>canvas.capturestream.enabled</code> to <code>true</code> to activate."
+            },
+            "ie": {
+              "version_added": false
+            },
+            "ie_mobile": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "requestFrame": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/CanvasCaptureMediaStream/requestFrame",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": null
+            },
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "41",
+              "notes": "Disabled by default; set the preference <code>canvas.capturestream.enabled</code> to <code>true</code> to activate."
+            },
+            "firefox_android": {
+              "version_added": "34"
```

Comment:
This looks dodgy. I think this was a bad cut and paste, put "41" and the flag there, too.